src/coreComponents/unitTests/constitutiveTests/testCO2BrinePVTModels.cpp

```diff
@@ -364,7 +364,8 @@ std::unique_ptr< MODEL > makePVTFunction( string const & filename,
       pvtFunction = std::make_unique< MODEL >( strs[1],
                                                strs,
                                                componentNames,
-                                               componentMolarWeight );
+                                               componentMolarWeight,
+                                               true ); // print PVT tables
```

Comment:
Do you then want to add test on the file generated ? Test that you can read back and compared to memory value ?

Reply:
not really to be honest